src/components/modal/Modal.vue

```diff
@@ -27,7 +28,8 @@
                 <slot v-else/>
                 <button
                     type="button"
-                    v-if="showX && !animating"
+                    v-if="showX"
+                    v-show="!animating"
```

Comment:
I use `v-show` instead of `v-if` because I want to be able to focus on the close button when using `trap-focus`. When we use `v-if`, the element does not exist within the `bind` function of the `trapFocus` directive.